vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -991,6 +1181,107 @@
                     });
                 });
                 
+                function getSpaceById(spaceId, callback) {
```

Comment:
get space by id should use an api call that actually gets the space by its id, not search all and then iterate over all!